en/health/centreon-agent.md

```diff
@@ -0,0 +1,511 @@
+
+- Configure all other Agents (installed next to Pollers, Remote Servers, MAP, etc.) as Gateway Clients
+
+#### Proxy Configuration
+
+If you have a proxy access configured on the host machine, copy the proxy settings to the `/etc/centreon-agent/centreon-agent.yml` file under **output**:
+
+```yaml
+output:
+  token: [your-token]
+  proxy_url: [your-proxy-address]:[your-desired-port]
+  proxy_ssl_insecure: [true|false]
+```
+
+Example:
+
+```yaml
+output:
+  token: aaaa-aaaa-aaaa-aaaa
+  proxy_url: http//proxy.local.net:3128
+  proxy_ssl_insecure: false
+```
+
+You then need to restart the Agent:
+
+```
+systemctl restart centreon-agent.service
+```
+
+#### Gateway Configuration
+
+- Gateway Server: copy the following code to the `/etc/centreon-agent/centreon-agent.yml` file of the Agent that will act as a Gateway server. To strengthen the security of communications between the gateway client and the gateway server, you can define an authentication token (`auth-token`), i.e. the character string you want (this is not the same token as the one you used to configure the `centreon-agent.yml` file).
+
+    ```yaml
+    gateway:
+      enable: true
+      listen_port: [listening-port]
+      auth_token: [your-gateway-token]
+    ```
+    
+    Example:
+
+    ```yaml
+    gateway:
+      enable: true
+      listen_port: 54321
+      auth_token: azerty1234
+    ```
+
+    You then need to restart the Agent
+
+    ```
+    systemctl restart centreon-agent.service
+    ```
+
+- Gateway Client
+
+    In a Gateway configuration, the Gateway Client delegates the configuration of its main token to the Gateway Server (since only the latter communicates with our platform).
+    As a consequence, the `token` line needs to be commented with the yaml comment operator “#”.
+    If you have defined an authentication token (`auth_token`) on the gateway server, you need to add it to the configuration of the gateway client too. 
+
+    ```yaml
+    output:
+    #token: [your-token]
+      gateway:
+        url: http://[gateway-server-ip-address]:[listening-port]
+        auth_token: [your-gateway-token]
+    ```
+
+    Example:
+
+    ```yaml
+    output:
+    #token: aaaa-aaaa-aaaa-aaaa
+      gateway:
+        url: http://172.28.6.145:54321
+        auth_token: azerty1234
+    ```
+
+    You then need to restart the Agent
+
+    ```
+    systemctl restart centreon-agent.service
+    ```
+
+### Enabling the Collection of Centreon Logs
+
+Starting from version 2 and up of the **centreon-agent**, logs generated by the monitored Centreon component can be collected. 
+
+To define which logs must be collected, you need to create yml configuration files in the following folder: `/etc/centreon-agent/conf.d`.
+To collect a specific log, the configuration file must contain the following arguments: path, type and pattern of the target log file. 
+You can have several configuration files - each file is parsed and its target log files are added to the collection. 
```

Comment:
We should add a real example what these three parameters looks like (taken from the examples for instance)